kernel/cooking.ml

```diff
@@ -185,32 +185,37 @@ let cook_constr { Opaqueproof.modlist ; abstract = (vars, subst, _) } c =
   let hyps = abstract_context hyps in
   abstract_constant_body (expmod c) hyps
 
+let discharge_abstract_universe_context subst abs_ctx auctx =
```

Comment:
The copy of this function in Lib should be deleted.

Reply:
It's used by Classes though?

Reply:
So export it from cooking

Reply:
Classes seems to reimplement its own cooking part. I am not sure I want to export internal stuff that is simple enough to be done locally...

Reply:
discharge_abstract_universe_context is not simple
If it was really internal there wouldn't be a copy in lib
Don't leave copies around after moving code